test/trixi_include.jl

```diff
@@ -127,3 +127,74 @@
         end
     end
 end
+
+@trixi_testset "`trixi_include_changeprecision`" begin
+    @trixi_testset "Basic" begin
+        example = """
+            x = 4.0
+            y = zeros(3)
+            """
+
+        filename = tempname()
```

Comment:
Why handle tempfile generation and deletion manually instead of using the automatic cleanup of a `do` block with [`Base.Filesystem.mktemp`](https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.mktemp-Tuple{Function,%20AbstractString})?

Reply:
Why did you do it when you updated these files last year? 🙃

Reply:
Updated all tests to use `mktemp`.

Reply:
> Why did you do it when you updated these files last year? 🙃

🙈 Good question, I do not remember. I assume mindless copy pasta.

Reply:
Thanks!